packages/database/src/migrations/20210511024543-CreateDashboardItemsTableLayout-modifies-schema.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  await db.runSql(`
+    CREATE TABLE dashboard_item (
+      id TEXT PRIMARY KEY,
+      code TEXT NOT NULL UNIQUE,
+      config JSONB NOT NULL DEFAULT '{}',
+      permission_groups TEXT[] NOT NULL CONSTRAINT permission_groups_not_empty CHECK (permission_groups <> '{}'),
+      report_code TEXT NOT NULL,
+      legacy BOOLEAN NOT NULL DEFAULT 'false'
+    );
+  `);
+  await db.runSql(`
+    CREATE TABLE legacy_report (
+      id TEXT PRIMARY KEY,
+      "dataBuilder" TEXT,
+      "dataBuilderConfig" JSONB,
+      "dataServices" JSONB DEFAULT '[{"isDataRegional": true}]'
```

Comment:
I'd like this to be a cleaner table too, so let's add `code` and have `id` do its `uuid` thing (already mentioned), but also change the columns to snake case for consistency